site/app/templates/grading/electronic/ta_status/Scatterplot.twig

```diff
@@ -14,7 +14,7 @@
             <option value="scores">Autograding Total Score</option>
         </select>
     </div>
-    <div id="myDiv4"></div>
+    <div id="myScatterplot5"></div>
```

Comment:
Can you remove the my* IDs throughout this file easily?

Reply:
Yes, naming updated in [6c4f366](https://github.com/Submitty/Submitty/pull/10687/commits/6c4f36650bc44bf2d4be1ec90f1ca42a498c2fb5)